charts/prefect-worker/templates/deployment.yaml

```diff
@@ -68,6 +68,12 @@ spec:
             - kubernetes
             - --pool
             - {{ .Values.worker.config.workPool | quote }}
+            {{- if .Values.worker.config.workQueues }}
```

Comment:
I don't think you need the `if` because a range over an empty list won't run (like a for-each over an empty array)

Reply:
nice !